src/controllers/user.controller.js

```diff
@@ -130,9 +130,9 @@ class UserController {
     try {
       const theUser = await UserService.findOne(req.body.email, '');
       if (theUser) {
-        return res.status(404).send({
-          status: 404,
-          message: `Cannot register User with the id ${req.body.email} which is already in use`
+        return res.status(400).send({
+          status: 400,
```

Comment:
A better status code for a conflict is 409

Reply:
Fixed, please check again